app/exports/candidate_journey_tracking_export.yml

```diff
@@ -0,0 +1,157 @@
+    description: Date the first completed reference was requested
+    example: 2020-11-01T00:00:00+00:00
+
+  completed_reference_1_received_at:
+    type: string
+    format: date-time
+    description: Date the first completed reference was received
+    example: 2020-11-01T00:00:00+00:00
+
+  completed_reference_2_requested_at:
+    type: string
+    format: date-time
+    description: Date the second completed reference was requested
+    example: 2020-11-01T00:00:00+00:00
+
+  completed_reference_2_received_at:
+    type: string
+    format: date-time
+    description: Date the second completed reference was received
+    example: 2020-11-01T00:00:00+00:00
+
+  reference_reminder_email_sent:
+    type: string
+    format: date-time
+    description: Date the earliest reference reminder email was sent
+    example: 2020-11-01T00:00:00+00:00
+
+  new_reference_request_email_sent:
+    type: string
+    format: date-time
+    description: Date the earliest new reference request email was sent
+    example: 2020-11-01T00:00:00+00:00
+
+  new_reference_added:
+    type: string
+    format: date-time
+    description: Date a new reference was added
+    example: 2020-11-01T00:00:00+00:00
+
+  rbd_date:
+    type: string
+    format: date-time
+    description: Date the application choice will be rejected by default
+    example: 2020-11-01T00:00:00+00:00
+
+  rbd_reminder_sent:
+    type: string
+    format: date-time
+    description: Date the earliest RBD reminder email was sent
+    example: 2020-11-01T00:00:00+00:00
+
+  application_rbd:
+    type: string
+    format: date-time
+    description: Date the application choice was rejected by default
+    example: 2020-11-01T00:00:00+00:00
+
+  provider_decision:
+    type: string
+    format: date-time
+    description: Date the provider gave an offer or rejected an application
+    example: 2020-11-01T00:00:00+00:00
+
+  offer_made:
+    type: string
+    format: date-time
+    description: Date the provider gave an offer
+    example: 2020-11-01T00:00:00+00:00
+
+  dbd_date:
+    type: string
+    format: date-time
+    description: Date the application choice will be declined by default
+    example: 2020-11-01T00:00:00+00:00
+
+  dbd_reminder_sent:
+    type: string
+    format: date-time
+    description: Date the earliest DBD reminder email was sent
+    example: 2020-11-01T00:00:00+00:00
+
+  candidate_decision:
+    type: string
+    format: date-time
+    description: Date the candidate accepted or declined an offer
+    example: 2020-11-01T00:00:00+00:00
+
+  offer_declined:
+    type: string
+    format: date-time
+    description: Date the candidate declined an offer
+    example: 2020-11-01T00:00:00+00:00
+
+  offer_accepted:
+    type: string
+    format: date-time
+    description: Date the candidate accepted an offer
+    example: 2020-11-01T00:00:00+00:00
+
+  pending_conditions:
+    type: string
+    format: date-time
+    description: Date the candidate accepted an offer
```

Comment:
This feels like a bug. For now I have documented the actual behaviour (in case I have misunderstood something). But this is exactly the same as `offer_accepted`. 

https://github.com/DFE-Digital/apply-for-teacher-training/blob/f9b37c806e57cac171f7f74a10c6176bd696686e/app/services/support_interface/candidate_journey_tracker.rb#L92